api/api.yaml

```diff
@@ -1672,6 +1672,75 @@ paths:
             application/problem+json:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
+  /projects/{projectIdOrName}/connections:create:
```

Comment:
As mentioned in my [Slab comment](https://ampersand.slab.com/posts/support-additional-auth-types-sp8yya06#thread-mmj3rdly), how about `generate` as the verb?

Reply:
that works